pkg/upstream/http.go

```diff
@@ -153,6 +153,7 @@ func setProxyDirector(proxy *httputil.ReverseProxy) {
 		// use RequestURI so that we aren't unescaping encoded slashes in the request path
 		req.URL.Opaque = req.RequestURI
 		req.URL.RawQuery = ""
+		req.URL.ForceQuery = false
```

Comment:
Is this needed in `setProxyUpstreamHostHeader` too?

Reply:
Out of curiosity, where is this getting set to `true` before this point?

Reply:
Good catch! Yes it will want to be set there too! In fact, tbh, looking at this, the whole point of these is that they are meant to be chained, let me rework this a bit